tests/data/bad_api.py

```diff
@@ -0,0 +1,62 @@
+class A:
+    pass
+
+
+class B:
+    @staticmethod
+    def build_key():
+        pass
+
+
+class C:
+    @staticmethod
+    def build_entity():
+        pass
+
+
+class D:
+    @staticmethod
+    def build_key():
+        pass
+
+    @staticmethod
+    def build_entity():
+        pass
+
+
+class E:
+    @staticmethod
+    def transform_records_list():
+        pass
+
+    @staticmethod
+    def build_key():
+        pass
+
+    @staticmethod
+    def build_entity():
+        pass
+
+
+class Good:
+    class_name = "good"
+    target_id_concept = "CONCEPT|FOO|TARGET_SERVICE_ID"
+
+    @staticmethod
+    def transform_records_list(records_list):
+        pass
+
+    @staticmethod
+    def build_key(record):
+        pass
+
+    @staticmethod
+    def build_entity(record, key, get_target_id_from_record):
+        pass
```

Comment:
Should we keep this in `bad_api.py`? Isn't this a good template example?

Reply:
It's test data for testing bad apis